### ISSUE TITLE
Setting GUI Position

### ISSUE THREAD

**Issue by Ichisich** [RamonUnch/AltSnap#151]
A little nitpick.

AltSnap's setting window always appears in the top left corner, diametral to the system tray where the AltSnap symbol resides.
It would improve handling to set the default position in the bottom right corner or letting it remember its last position.

**Comment by RamonUnch** [comment]
True, AltSnap just lets Windows decide where the dialog appears, it would be more intuitive to show it in the bottom right corner, or maybe in the corner where the cursor is (if someone sets up his taskbar to be on the top/side...
I will implement this.

**Comment by RamonUnch** [comment]
Sorry the above ia actually a 32b build (not that it matters unless you are using FancyZones integartion.
[AltSnap1.50test24_x64.zip](https://github.com/RamonUnch/AltSnap/files/8095601/AltSnap1.50test24_x64.zip)

**Comment by Ichisich** [comment]
Works flawlessly with all four task-bar locations.
